content/material_and_questions.md

```diff
@@ -13,23 +19,26 @@
 | 2026-2027 | John                                          | Narrative |
 | 2027-2028 | Hebrews, 1 Peter, 2 Peter                     | Epistle   |
 
-## Question Writing
-
-### Preparation of Questions
-
-### Question Types
+Question Types
+: The Type that a particular quiz question is.
 
-#### Interrogative
+Question Sub-Types
+: A different / more-specific question type that falls under the main type. For the purposes of the rule book, any use of the term "Type" will assume to include all sub-types of that type.
 
-#### Multiple Answer
+Quiz Question
+: A quiz question consists of a Question Type, Question, and Answer. 
 
-#### Reference
+Interrogative Words
+: The single, interrogative words that must be inserted at either the beginning or the end of every [ Interrogative, Multiple Answer, Chapter Reference, Chapter Verse Reference ] question. The exhaustive list of these words is: who, whom, whose, what, why, where, when, which, how.
 
-#### Quote
+Unique Word
+: A word appearing only once in the material or a word that occurs multiple times, but only in 1 verse in the material. Also known as "global unique words."  
 
-#### Finish
+Unique Phrases
+: 2 or 3 consecutive words, the sequence of which is only found once in the material. While unique words have the "occurring more than once but in the same verse" corollary, this concept does not exist for 2-3 word phrases. 
```

Comment:
I added explicit language on unique phrases that doesn't allow for a phrase to exist more than once, but only in the same verse, to be a valid unique phrase. Thoughts?

Reply:
I agree this clarification is substantively identical to the original rulebook, but I wasn't aware that the original rulebook said this. 

I definitely have written questions based on "unique phrases" that occur multiple times in one verse... the rule really should change, because it prohibits us from asking many good standard MA questions, for example from Matt. 12:33, "Make a tree what?"

Reply:
hmmmm, you're right, I have to.

Reply:
I recommend we write v1 such that we have "unique" meaning "doesn't happen outside the current verse", which I think is func equiv to what IBQ does right now.